flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "skip-arches": ["aarch64", "arm"]
```

Comment:
What fails to build on these architectures?

Reply:
Here is the log from build https://flathub.org/builds/#/builders/2/builds/3002

Reply:
Should be taken to upstream, maybe latest Rust introduced some incompatibilities.